include/swift/AST/Decl.h

```diff
@@ -2282,8 +2289,8 @@ class ValueDecl : public Decl {
   /// \sa getFormalAccess
   /// \sa isAccessibleFrom
   AccessScope
-  getFormalAccessScope(const DeclContext *useDC = nullptr,
-                       bool respectVersionedAttr = false) const;
+  getFormalAccessScope(const DeclContext *useDC=nullptr,
+                       bool isUsageFromInline=false) const;
```

Comment:
Nitpick: Why remove the spaces?

Reply:
Also this name isn't really correct. The formal access scope doesn't change when used from an inline function; it changes when it's only being used as a symbol and not as part of API.

Reply:
I renamed this in the other PR that got merged already.